### PR TITLE
[Prim][PIR]Set rsqrt as primitive op

### PR DESCRIPTION
**Pull request by cyber-pioneer**
### PR types
Others

### PR changes
Others

### Description
Pcard-66975
Set rsqrt as primitive op for better performance in cinn, temporarily.

### CODE REVIEW

**Review by Charles-hit — APPROVED** [review]

File: test/prim/pir_prim/test_auto_recompute.py [hunk 1]
```diff
@@ -153,11 +153,11 @@ def test_auto_recompute(self):
                 atol=TOLERANCE[self.dtype]["atol"],
                 rtol=TOLERANCE[self.dtype]["rtol"],
             )
-            forward_ops = recompute_program.global_block().ops[:14]
-            backward_ops = recompute_program.global_block().ops[14:]
-            saved_values = forward_ops[9].results()[0]
+            forward_ops = recompute_program.global_block().ops[:13]
+            backward_ops = recompute_program.global_block().ops[13:]
+            saved_values = forward_ops[10].results()[0]
```
Comment (jeff41404) [inline test/prim/pir_prim/test_auto_recompute.py:158]: Is this kind of hard coding of numbers making maintenance difficult, and are there any better methods?

File: test/prim/pir_prim/test_auto_recompute_dy2static.py [hunk 2]
```diff
@@ -127,9 +127,9 @@ def test_auto_recompute(self):
             forward_ops = actual_program.global_block().ops[:15]
             mid_ops = actual_program.global_block().ops[15:18]
             backward_ops = actual_program.global_block().ops[18:]
-            saved_values = forward_ops[9].results()[0]
+            saved_values = forward_ops[10].results()[0]
```
Comment (jeff41404) [inline test/prim/pir_prim/test_auto_recompute_dy2static.py:130]: same issue as above

File: paddle/fluid/primitive/primitive.yaml [hunk 3]
```diff
@@ -3,6 +3,7 @@
 - multiply
 - divide
 - elementwise_pow
+- rsqrt
```
Comment (jeff41404) [inline paddle/fluid/primitive/primitive.yaml:6]: Does vjp rule of `rsqrt` need to be defined?

Comment (cyber-pioneer) [reply paddle/fluid/primitive/primitive.yaml:6]: yes, to do later

File: test/prim/pir_prim/test_auto_recompute.py (hunk 1 above)
Comment (cyber-pioneer) [reply test/prim/pir_prim/test_auto_recompute.py:158]: This op is the dividing point in recompute strategy, and it is hard to determine in advance manual.

File: test/prim/pir_prim/test_auto_recompute_dy2static.py (hunk 2 above)
Comment (cyber-pioneer) [reply test/prim/pir_prim/test_auto_recompute_dy2static.py:130]: same reply as above

File: test/prim/pir_prim/test_auto_recompute.py (hunk 1 above)
Comment (Charles-hit) [reply test/prim/pir_prim/test_auto_recompute.py:158]: TODO: add interface to maintenance forward op end index information in to static.

File: test/prim/pir_prim/test_auto_recompute_dy2static.py (hunk 2 above)
Comment (Charles-hit) [reply test/prim/pir_prim/test_auto_recompute_dy2static.py:130]: TODO: remove it in next pr.

**Review by jeff41404 — APPROVED** [review]

**Pull request merged**